content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md

```diff
@@ -0,0 +1,30 @@
+---
+weight: 60
+title: Release 10.13.0
+layout: redirect
+---
+
+Cumulocity IoT Edge Release 10.13, includes the following improvements, fixes and known issues.
+
+Download the Cumulocity IoT Edge version 10.13 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Changing the hostname
+
+You can now change the hostname of your Edge appliance using the user interface. For more information, see [Configuring the hostname using the UI](https://cumulocity.com/guides/edge/configuration/#configuring-the-hostname).
+
+### Configuring the time synchronization
+
+You can now configure the time synchronization of your Edge appliance using the user interface. For more information, [Configuring the time synchronization using the UI](https://cumulocity.com/guides/edge/configuration/#configuring-the-time-synchronization-using-the-ui).
```

Comment:
For more information, > For more information, see

Reply:
Done.